src/main/java/com/neroreflex/plugins/HackerNews.java

```diff
@@ -31,64 +31,62 @@
  */
 public final class HackerNews extends Trancio {
 
-    private int interval = 30000; //tempo tra le chiamate all'api in millisecondi
+    private int interval = (int)2.16e+7; // (= 6h) tempo tra le chiamate all'api in millisecondi
```

Comment:
Conviene usare `Duration.ofHours(6).toMillis()`
In un int ci starebbe, ma siccome `toMillis` ritorna `long` tanto vale cambiare il tipo di interval.